backend-tests/tests/test_rbac_v2.py

```diff
@@ -1912,7 +1912,7 @@ def test_manage_releases(self, clean_mongo, test_case):
         api_client_v2.with_auth(tenant.users[0].token)
         r = api_client_v2.call("GET", deployments_v2.URL_RELEASES)
         assert r.status_code == 200
-        all_releases = r.json()
+        all_releases = [x for x in r.json() if not x['name'].startswith('mender-demo-artifact')]
```

Comment:
```suggestion
        excluded_artifact_prefix = 'mender-demo-artifact'
        all_releases = [x for x in r.json() if not x['name'].startswith(excluded_artifact_prefix)]
```
feel free to ignore. bonus points: `excluded_artifact_prefixes` (array)